x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/ChoroplethMap/ChoroplethToolTip.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { asTime, asInteger } from '../../../../../utils/formatters';
+import { fontSizes } from '../../../../../style/variables';
+
+export const ChoroplethToolTip: React.SFC<{
+  name?: string;
+  value?: number;
+  docCount?: number;
```

Comment:
Can we make all of these required so we don't have to make the `null` checks inside the tooltip?
```js
  name: string;
  value: number;
  docCount: number;
```